From b32a169b079a67629304c146aad65c2d79acfe01 Mon Sep 17 00:00:00 2001
From: Zdenek Kabelac <zkabelac@redhat.com>
Date: Mon, 15 Jan 2018 16:26:34 +0100
Subject: [PATCH 23/25] tests: longer startup timeout for daemons with valgrind

It's getting noticable somewhat slower...

(cherry picked from commit 5baf2de8986ceeb353c39ccf0503f9fefcb6eb92)
---
 test/lib/aux.sh | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/test/lib/aux.sh b/test/lib/aux.sh
index 6bc7bd4..f30dfdd 100644
--- a/test/lib/aux.sh
+++ b/test/lib/aux.sh
@@ -180,8 +180,8 @@ prepare_clvmd() {
 	LVM_LOG_FILE_EPOCH=CLVMD LVM_LOG_FILE_MAX_LINES=1000000 LVM_BINARY=$(which lvm) $run_valgrind clvmd -Isinglenode -d 1 -f &
 	echo $! > LOCAL_CLVMD
 
-	for i in {1..100} ; do
-		test "$i" -eq 100 && die "Startup of clvmd is too slow."
+	for i in {200..0} ; do
+		test "$i" -eq 0 && die "Startup of clvmd is too slow."
 		test -e "$CLVMD_PIDFILE" && test -e "${CLVMD_PIDFILE%/*}/lvm/clvmd.sock" && break
 		echo -n .
 		sleep .1
@@ -205,8 +205,8 @@ prepare_dmeventd() {
 	echo $! > LOCAL_DMEVENTD
 
 	# FIXME wait for pipe in /var/run instead
-	for i in {1..100} ; do
-		test "$i" -eq 100 && die "Startup of dmeventd is too slow."
+	for i in {200..0} ; do
+		test "$i" -eq 0 && die "Startup of dmeventd is too slow."
 		test -e "${DMEVENTD_PIDFILE}" && break
 		echo -n .
 		sleep .1
@@ -230,8 +230,8 @@ prepare_lvmetad() {
 	$run_valgrind lvmetad -f "$@" -s "$TESTDIR/lvmetad.socket" \
 		${LVM_TEST_LVMETAD_DEBUG_OPTS--l all} &
 	echo $! > LOCAL_LVMETAD
-	for i in {1..100} ; do
-		test "$i" -eq 100 && die "Startup of lvmetad is too slow."
+	for i in {200..0} ; do
+		test "$i" -eq 0 && die "Startup of lvmetad is too slow."
 		test -e "$TESTDIR/lvmetad.socket" && break
 		echo -n .
 		sleep .1;
@@ -280,8 +280,8 @@ prepare_lvmpolld() {
 	echo -n "## preparing lvmpolld..."
 	$run_valgrind lvmpolld -f "$@" -s "$TESTDIR/lvmpolld.socket" -B "$TESTDIR/lib/lvm" -l all &
 	echo $! > LOCAL_LVMPOLLD
-	for i in {1..100} ; do
-		test "$i" -eq 100 && die "Startup of lvmpolld is too slow."
+	for i in {200..0} ; do
+		test "$i" -eq 0 && die "Startup of lvmpolld is too slow."
 		test -e "$TESTDIR/lvmpolld.socket" && break
 		echo -n .;
 		sleep .1;
-- 
1.8.3.1

